### PR TITLE
fix issue 89

### LINKED ISSUE #89: Need way of updating db rather than creating from scratch
If we have two repos that contribute to the same database, then it's hard to include tables from both repos.

For instance, if we have two repositories that both create tables in the open_data database (e.g. one repo that ETLs ONS data to the platform, and another than ETLs travel time data), then you can add either the ONS data  or the travel time data to the glue catalogue, but you can't easily have both.

### LINKED ISSUE #117: Allow user to add a table to existing database
If you have an existing database, it's hard at the moment to add an additional table without tearing down the whole thing and recreating it

### PR DESCRIPTION
**Pull request by RobinL**
Closes #89  Closes #117

### CODE REVIEW

File: etl_manager/meta.py [hunk 1]
```diff
@@ -644,6 +645,32 @@ def create_glue_database(self, delete_if_exists=False):
...
+            _glue_client.update_database(**db)
+
+        for tab in self._tables:
+            try:
+                _glue_client.get_table(Name=tab.name, DatabaseName=self.name)
+                table_exists = True
+            except _glue_client.exceptions.EntityNotFoundException:
+                table_exists = False
+
+            if not table_exists:
```
Comment (isichei) [inline etl_manager/meta.py:665]: Might be worth doing the below to reduce duplicate code
```python
if (not table_exists) or (table_exists and update_tables_if_exist):
```

File: etl_manager/meta.py [hunk 2]
```diff
@@ -756,3 +783,16 @@ def read_database_folder(folderpath):
         tm = read_table_json(table_file_path, database=db)
         db.add_table(tm)
     return db
+
+def get_existing_database_from_glue_catalogue(database_name):
```
Comment (isichei) [inline etl_manager/meta.py:787]: I would add a docstring here making it clear that it returns a DatabaseMeta Object without the tables (regardless if the tables existed or not)

File: etl_manager/meta.py (hunk 1 above)
Comment (RobinL) [reply etl_manager/meta.py:665]: they two different things methods on the `_glue_client` - one calls `create_table` and the other `update_table`

File: etl_manager/meta.py (hunk 2 above)
Comment (RobinL) [reply etl_manager/meta.py:822]: Done

**Pull request merged**